src/legacy/core_plugins/kibana/public/visualize/embeddable/visualize_embeddable.ts

```diff
@@ -152,26 +175,42 @@ export class VisualizeEmbeddable extends Embeddable  {
       cssClass: `panel-content panel-content--fullWidth`,
       // The chrome is permanently hidden in "embed mode" in which case we don't want to show the spy pane, since
```

Comment:
Please remove those two lines of comments, since they are not valid anymore. I guess I forgot to remove them in the appropriate PR.